integration-test/utils/datafactory/schemas/codeofconduct.schema.ts

```diff
@@ -0,0 +1,41 @@
+import { pageSchema } from './page.schema';
+import { linkSchema } from './link.schema';
+
+export const codeofconductSchema = {
+  $ref: '#/definitions/pageSchema',
+  definitions: {
+    codeofconductSchema: {
+      type: 'object',
+      properties: {
+        id: {
+          type: 'string',
+          minLength: 1,
+        },
+        page: {
+          page: { ...pageSchema.definitions.pageSchema },
+        },
+        items: {
```

Comment:
In `items` property there is no type of data provided. So, in this case it will not give you an error if `items` includes something else instead of array. 
<img width="372" alt="Screenshot 2025-01-28 at 21 44 34" src="https://github.com/user-attachments/assets/29f11ff8-813f-4b58-b178-9b18417eb70b" />